### DISCUSSION TITLE
Adding bootstrap css causes gridlines on p-table to disappear

### DISCUSSION THREAD

**Discussion by workvact** [primefaces/community#178] [PrimeNG] [answered]
I asked another question related to this issue and was answered here: https://github.com/orgs/primefaces/discussions/153
Uncommenting the bootstrap import in global_styes.css of the stack blitz fork in the answer (https://stackblitz.com/edit/stackblitz-starters-sq8d71?file=src%2Fglobal_styles.css) caused the grid lines disappear. I'm trying to introduce PrimeNG in to a project that is already using bootstrap. What is the correct way to add bootstrap reference if PrimeNg is used?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Bootstrap and primeflex classes are different. They may sometimes collide and one of them overrides, because of that you can find mismatch in styles of both libraries.

Your example works with bootstrap after a refactor, all you need to do is refactor your css imports please see the example https://stackblitz.com/edit/stackblitz-starters-sq8d71?file=src%2Fglobal_styles.css,src%2Fapp.component.html

For the table styling with bootstrap, I suggest you to use tableStyleClass property of the table.

You can either import library styles to your main css file or styles array in angular.json file.

**Reply by workvact** [reply]
Thank you.
